src/components/apps/queries.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * react-query functions for Apps.
+ *
+ * @author psarando
+ */
+import { useQuery } from "react-query";
+
+import {
+    getReferenceGenomes,
+    REFERENCE_GENOMES_QUERY_KEY,
+} from "serviceFacades/referenceGenomes";
+
+import { stableSort } from "@cyverse-de/ui-lib";
+
+export function useReferenceGenomes(enabled, onSuccess) {
+    return useQuery({
+        queryKey: [REFERENCE_GENOMES_QUERY_KEY, { deleted: false }],
+        queryFn: getReferenceGenomes,
+        config: {
+            enabled,
+            onSuccess: (resp) => {
+                const genomes = resp?.genomes || [];
+                onSuccess(
+                    stableSort(genomes, (a, b) => a.name.localeCompare(b.name))
+                );
+            },
+            onError: (e) => {
+                console.error(e);
```

Comment:
This is how it was before in the app launch form, since the reference genome select fields will display an inline error if they are not loading but have no options to display:
https://github.com/cyverse-de/sonora/blob/ed97e9fa7a6583051ee039a27a6f1b02cf090c6c/src/components/apps/launch/ReferenceGenomeSelect.js#L24-L27

Should we also show an announcer error message from here too?

Reply:
Ahh makes sense. Can we add some comments around the `console` message so that it is clear when we encounter this next time.`